### ISSUE TITLE
Aria-polite not in an effective place

### ISSUE THREAD

**Issue by tyrylu** [bitwarden/browser#1033]
Related to #1032 which was closed before my findings comments. The aria-live attribute should be moved to the div which gets injected to the document, so somewhere in notificationBar.ts where the div and iframe are created. It makes no difference whether the attribute is on the div or the iframe. Yes, using a role="alert" would probably be best, but adding it on the same elements results in some weird announcements and i have no idea whether adding it somewhere using some setTimeout inside of the iframe would have the effect which is texpected, would have to be tried with NVDA or similar...

**Comment by kspearrin** [comment]
Sorry, I misunderstood that notification you were referring to in the original issue. See https://github.com/bitwarden/browser/commit/3e1e05ab4ffabbf180972650818a3ae3468dbdfb

Is that better now?

**Comment by tyrylu** [comment]
Yes, thanks, closing this one because it accomplished its purpose.
